Packages/com.unity.multiplayer.samples.coop/Utilities/Net/RNSM/CustomNetStatsMonitorConfiguration.asset.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
Curious if this should live instead inside of our project, and not in the utilities package.

Oh and want to give the last 2 configurations a label too? It would make the whole graph read cleaner.

Reply:
2 labels: done
utilities: this config is used inside the RNSM prefab we have and this prefab is in utilities as well. It's not super customized to boss room, it's really more "better defaults" I would have used for RNSM. Users are free to use their own as well. It's just easier to use combined with our prefab.